tests/queries/0_stateless/03130_generate_snowflake_id.sql

```diff
@@ -0,0 +1,11 @@
+SELECT bitShiftLeft(toUInt64(generateSnowflakeID()), 52) = 0;
```

Comment:
Let's rename the test + reference file to 03130_generateSnowflakeId.sql/reference.